src/module.ts

```diff
@@ -140,8 +140,20 @@ export default defineNuxtModule<NuxtI18nOptions>({
      */
 
     // for public
+    // @ts-expect-error generated type
     nuxt.options.runtimeConfig.public.i18n = defu(nuxt.options.runtimeConfig.public.i18n, {
       baseUrl: options.baseUrl,
+      defaultLocale: options.defaultLocale,
+      defaultDirection: options.defaultDirection,
+      strategy: options.strategy,
+      lazy: options.lazy,
+      rootRedirect: options.rootRedirect,
+      routesNameSeparator: options.routesNameSeparator,
+      defaultLocaleRouteNameSuffix: options.defaultLocaleRouteNameSuffix,
+      skipSettingLocaleOnNavigate: options.skipSettingLocaleOnNavigate,
+      differentDomains: options.differentDomains,
+      trailingSlash: options.trailingSlash,
+      configLocales: options.locales,
```

Comment:
Hmm 🤔 
I would also prefer `locales` instead of `configLocales` if possible.
It seems that `locales` have been introduced by this PR.
https://github.com/nuxt-modules/i18n/pull/2446

In v9, we should also be organized runtimeConfig.
`locales` should be `domainLocales` and `configLocales` should be `locales`.

@BobbieGoede 
What do you think?

Reply:
>In v9, we should also be organized runtimeConfig.
locales should be domainLocales and configLocales should be locales.

Agreed, that sounds good!

Reply:
Thanks! Okay, I'll merge this PR!